src/Microsoft.OpenApi.OData.Reader/Common/Utils.cs

```diff
@@ -270,5 +271,28 @@ internal static void AddCustomAttributesToExtensions(this IDictionary<string, IO
 
             return false;
         }
+
+        /// <summary>
+        /// Removes the default namespace from a string value.
+        /// </summary>
+        /// <param name="value">The target string value.</param>
+        /// <param name="settings">Convert settings.</param>
+        /// <returns>The string value with the namespace removed.</returns>
+        internal static string RemoveDefaultNamespace(this string value, OpenApiConvertSettings settings)
+        {
+            CheckArgumentNullOrEmpty(value, nameof(value));
+            CheckArgumentNull(settings, nameof(settings));
+
+            if (string.IsNullOrEmpty(settings.DefaultNamespace))
+            {
+                return value;
+            }
+
+            string defaultNamespace = settings.DefaultNamespace.EndsWith(".")
+                ? settings.DefaultNamespace
+                : settings.DefaultNamespace + ".";
+
+            return value.Replace(defaultNamespace, "");
```

Comment:
```suggestion
            return value.StartsWith(defaultNamespace, StringComparison.OrdinalIgnoreCase) ? value.Remove(0, defaultNamespace.Length) : value;
```

Reply:
This extension method has been removed in the latest commit.